src/bin/demo-addition.ts

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+
+import process from 'process';
+import NumPair from '../lib/NumPair';
+
+function main(argv = process.argv): number {
+  const nums = new NumPair(parseInt(argv[2]), parseInt(argv[3]));
+  const sum = nums.num1 + nums.num2;
+  console.log(nums.num1 + ' + ' + nums.num2 + ' = ' + sum);
```

Comment:
Oh if you are doing this, then there's console.log being used.

But CLI commands should be using process.stdout not console.log.

Reply:
Try to add `process.stdout.write` as well. Then your tests should involve the `process.stdout.write` not `console.log`.